docs/core/deploying/acquisition-experiences.md

```diff
@@ -0,0 +1,173 @@
+﻿---
+title: .NET Core Acquisition Experiences
+description: .NET Core Acquisition Experiences
+author: jamshedd
+ms.author: jamshedd
+ms.date: 09/06/2019
+ms.custom: 
+---
+
+# .NET Core Acquisition Experiences
+
```

Comment:
each article will need an intro of what they'll cover

Reply:
> I'd like to discuss these changes in more detail. I think that deployment and acquisition should be kept separate.
> 
> Perhaps we create an Installation node similar to what VS docs has and add the articles related to that under that new node, including the existing pre-reqs topics.
> 
> And then we have the deployment node for anything related to packing and publishing your app.

Putting together a flow that ties this together has been difficult, I think we have the flow now. I instead propose renaming the deployment node to Acquisition and Deployment. The index.md will start the topic, and both acquisition and deployment are separate pages/topics in the section.